users/models.py

```diff
@@ -5,11 +5,11 @@ class User(models.Model):
     last_name           = models.CharField(max_length=16)
     first_name          = models.CharField(max_length=32)
     email               = models.EmailField()
-    password            = models.CharField(max_length=256)
-    birthday            = models.DateField()
+    password            = models.CharField(max_length=256,default="a123456789")
+    birthday            = models.DateField(null=True,default="2020-12-31")
```

Comment:
```suggestion
    birthday            = models.DateField(null=True)
```
왜 특정 일자를 디폴트로 넣으시는거죠?

Reply:
get_or_create 방식으로 소셜로그인 구현하였으나
기존 회원가입 시 받아야 하는 정보들을 전부 받지 않는 경우 문제가 발생해서
카카오 회원가입시 받지 않는 정보에 대해서 디폴트 값을 주었습니다.

password 부분에 null=True를 주고 진행하였으나, 500에러가 발생하여
기존 방식으로 롤백하였습니다.

더욱 확실하게 익힌 뒤 다시 구현하겠습니다.